package.json

```diff
@@ -0,0 +1,60 @@
+{
+  "name": "audion",
+  "version": "1.0.0",
+  "description": "Traces web audio API calls.",
+  "main": "index.js",
+  "engines": {
+    "node": "14"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.14.6",
+    "@babel/plugin-transform-modules-commonjs": "^7.14.5",
+    "@pixi/jsdoc-template": "^2.6.0",
+    "babel-jest": "^27.0.6",
+    "eslint": "^7.30.0",
+    "eslint-config-google": "^0.14.0",
+    "husky": ">=6",
+    "jest": "^27.0.6",
+    "jest-puppeteer": "^5.0.4",
+    "jsdoc": "^3.6.7",
+    "lint-staged": ">=10",
+    "mustache": "^4.2.0",
```

Comment:
Why do we need an additional template library?

Reply:
Additional? I think it's the only template library depended on currently.

Reply:
Mustache might have additional values, but I thought JS template literal can do many things that Mustache does. I am just curious, and this is not a blocker.

Reply:
Kk. I'll revisit mustache.